{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.1",
	"cveMetadata": {
		"cveId": "CVE-2011-2887",
		"assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
		"assignerShortName": "mitre",
		"state": "PUBLISHED",
		"datePublished": "2011-07-27T20:00:00",
		"dateUpdated": "2024-08-06T23:15:31.607Z",
		"dateReserved": "2011-07-27T00:00:00"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
				"shortName": "mitre",
				"dateUpdated": "2017-08-28T12:57:01"
			},
			"descriptions": [
				{
					"lang": "en",
					"value": "IBM Lotus Symphony 3 before FP3 on Linux allows remote attackers to cause a denial of service (application crash) via a certain sample document."
				}
			],
			"affected": [
				{
					"vendor": "n/a",
					"product": "n/a",
					"versions": [
						{
							"status": "affected",
							"version": "n/a"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "text",
							"lang": "en",
							"description": "n/a"
						}
					]
				}
			],
			"references": [
				{
					"name": "lotus-symphony-document-dos(68889)",
					"tags": [
						"vdb-entry",
						"x_refsource_XF"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/68889"
				},
				{
					"name": "48936",
					"tags": [
						"vdb-entry",
						"x_refsource_BID"
					],
					"url": "http://www.securityfocus.com/bid/48936"
				},
				{
					"tags": [
						"x_refsource_CONFIRM"
					],
					"url": "http://www.ibm.com/support/docview.wss?uid=swg21505448"
				},
				{
					"tags": [
						"x_refsource_CONFIRM"
					],
					"url": "http://www.ibm.com/software/lotus/symphony/buzz.nsf/web_DisPlayPlugin?open&unid=9717F6F587AAA939852578D300404BCF&category=announcements"
				},
				{
					"tags": [
						"x_refsource_CONFIRM"
					],
					"url": "https://www-304.ibm.com/jct03001c/software/lotus/symphony/idcontents/releasenotes/en/readme_embedded_in_fixpack3_long.htm"
				},
				{
					"tags": [
						"x_refsource_CONFIRM"
					],
					"url": "http://www.ibm.com/software/lotus/symphony/idcontents/releasenotes/en/readme_fixpack3_standalone_long.htm"
				},
				{
					"name": "74163",
					"tags": [
						"vdb-entry",
						"x_refsource_OSVDB"
					],
					"url": "http://osvdb.org/74163"
				}
			],
			"datePublic": "2011-07-20T00:00:00",
			"x_legacyV4Record": {
				"CVE_data_meta": {
					"ASSIGNER": "cve@mitre.org",
					"ID": "CVE-2011-2887",
					"STATE": "PUBLIC"
				},
				"affects": {
					"vendor": {
						"vendor_data": [
							{
								"product": {
									"product_data": [
										{
											"product_name": "n/a",
											"version": {
												"version_data": [
													{
														"version_value": "n/a"
													}
												]
											}
										}
									]
								},
								"vendor_name": "n/a"
							}
						]
					}
				},
				"data_format": "MITRE",
				"data_type": "CVE",
				"data_version": "4.0",
				"description": {
					"description_data": [
						{
							"lang": "eng",
							"value": "IBM Lotus Symphony 3 before FP3 on Linux allows remote attackers to cause a denial of service (application crash) via a certain sample document."
						}
					]
				},
				"problemtype": {
					"problemtype_data": [
						{
							"description": [
								{
									"lang": "eng",
									"value": "n/a"
								}
							]
						}
					]
				},
				"references": {
					"reference_data": [
						{
							"name": "lotus-symphony-document-dos(68889)",
							"refsource": "XF",
							"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/68889"
						},
						{
							"name": "48936",
							"refsource": "BID",
							"url": "http://www.securityfocus.com/bid/48936"
						},
						{
							"name": "http://www.ibm.com/support/docview.wss?uid=swg21505448",
							"refsource": "CONFIRM",
							"url": "http://www.ibm.com/support/docview.wss?uid=swg21505448"
						},
						{
							"name": "http://www.ibm.com/software/lotus/symphony/buzz.nsf/web_DisPlayPlugin?open&unid=9717F6F587AAA939852578D300404BCF&category=announcements",
							"refsource": "CONFIRM",
							"url": "http://www.ibm.com/software/lotus/symphony/buzz.nsf/web_DisPlayPlugin?open&unid=9717F6F587AAA939852578D300404BCF&category=announcements"
						},
						{
							"name": "https://www-304.ibm.com/jct03001c/software/lotus/symphony/idcontents/releasenotes/en/readme_embedded_in_fixpack3_long.htm",
							"refsource": "CONFIRM",
							"url": "https://www-304.ibm.com/jct03001c/software/lotus/symphony/idcontents/releasenotes/en/readme_embedded_in_fixpack3_long.htm"
						},
						{
							"name": "http://www.ibm.com/software/lotus/symphony/idcontents/releasenotes/en/readme_fixpack3_standalone_long.htm",
							"refsource": "CONFIRM",
							"url": "http://www.ibm.com/software/lotus/symphony/idcontents/releasenotes/en/readme_fixpack3_standalone_long.htm"
						},
						{
							"name": "74163",
							"refsource": "OSVDB",
							"url": "http://osvdb.org/74163"
						}
					]
				}
			}
		},
		"adp": [
			{
				"providerMetadata": {
					"orgID": "af854a3a-2127-422b-91ae-364da2661108",
					"shortName": "CVE",
					"dateUpdated": "2024-08-06T23:15:31.607Z"
				},
				"title": "CVE Program Container",
				"references": [
					{
						"name": "lotus-symphony-document-dos(68889)",
						"tags": [
							"vdb-entry",
							"x_refsource_XF",
							"x_transferred"
						],
						"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/68889"
					},
					{
						"name": "48936",
						"tags": [
							"vdb-entry",
							"x_refsource_BID",
							"x_transferred"
						],
						"url": "http://www.securityfocus.com/bid/48936"
					},
					{
						"tags": [
							"x_refsource_CONFIRM",
							"x_transferred"
						],
						"url": "http://www.ibm.com/support/docview.wss?uid=swg21505448"
					},
					{
						"tags": [
							"x_refsource_CONFIRM",
							"x_transferred"
						],
						"url": "http://www.ibm.com/software/lotus/symphony/buzz.nsf/web_DisPlayPlugin?open&unid=9717F6F587AAA939852578D300404BCF&category=announcements"
					},
					{
						"tags": [
							"x_refsource_CONFIRM",
							"x_transferred"
						],
						"url": "https://www-304.ibm.com/jct03001c/software/lotus/symphony/idcontents/releasenotes/en/readme_embedded_in_fixpack3_long.htm"
					},
					{
						"tags": [
							"x_refsource_CONFIRM",
							"x_transferred"
						],
						"url": "http://www.ibm.com/software/lotus/symphony/idcontents/releasenotes/en/readme_fixpack3_standalone_long.htm"
					},
					{
						"name": "74163",
						"tags": [
							"vdb-entry",
							"x_refsource_OSVDB",
							"x_transferred"
						],
						"url": "http://osvdb.org/74163"
					}
				]
			}
		]
	}
}
